applications/RomApplication/python_scripts/rom_solver.py

```diff
@@ -19,19 +20,28 @@ def __init__(self, model, custom_settings):
         @classmethod
         def GetDefaultParameters(cls):
             default_settings = KratosMultiphysics.Parameters("""{
+                "solving_strategy" : "Galerkin",
                 "rom_settings": {
                     "nodal_unknowns": [],
-                    "number_of_rom_dofs": 0
+                    "number_of_rom_dofs" : 10
                 }
             }""")
             default_settings.AddMissingParameters(super().GetDefaultParameters())
             return default_settings
 
         def _CreateBuilderAndSolver(self):
             linear_solver = self._GetLinearSolver()
-            rom_parameters = self._ValidateAndReturnRomParameters()
-            builder_and_solver = KratosROM.ROMBuilderAndSolver(linear_solver, rom_parameters)
-            return builder_and_solver
+            rom_parameters, solving_strategy = self._ValidateAndReturnRomParameters()
+            available_solving_strategies = {
+                "Galerkin": KratosROM.ROMBuilderAndSolver, 
+                "LSPG": KratosROM.LSPGROMBuilderAndSolver,
+                "Petrov-Galerkin": KratosROM.PetrovGalerkinROMBuilderAndSolver
```

Comment:
```suggestion
                "galerkin": KratosROM.ROMBuilderAndSolver, 
                "least_squares_petrov_galerkin": KratosROM.LSPGROMBuilderAndSolver,
                "petrov_galerkin": KratosROM.PetrovGalerkinROMBuilderAndSolver
```
More Kratos-like naming.

Reply:
Besides, I'd put this dict in an auxiliary method so we can eventually use it in other places. Note that doing so also makes easier an eventual extension of it.